docs/table-partitioning-bucketing-sort-order.md

```diff
@@ -0,0 +1,285 @@
+| constant        | `{"type":"literal","value":10, "dataType": "integer"}`         | `new LiteralDTO.Builder().withValue("10").withDataType(Types.IntegerType.get()).build()`  | `10`                     | The integer literal `10`          |
+
+
+<Tabs>
+<TabItem value="Json" label="Json">
+
+```json
+{
+  "strategy": "hash",
+  "number": 4,
+  "funcArgs": [
+    {
+      "type": "field",
+      "fieldName": ["score"]
+    }
+  ]
+}
+```
+
+</TabItem>
+<TabItem value="java" label="Java">
+
+```java
+ new DistributionDTO.Builder()
+    .withStrategy(Strategy.HASH)
+    .withNumber(4)
+    .withArgs(FieldReferenceDTO.of("score"))
+    .build()
+```
+
+</TabItem>
+</Tabs>
+
+
+## Sort ordering
+
+To define a sorted order table, you should use the following three components to construct a valid sorted order table.
+
+- Direction. It defines in which direction Gravitino sorts the table. The default value is `ascending`.
+
+| Direction  | Description                                 | JSON   | Java                       |
+|------------|---------------------------------------------| ------ | -------------------------- |
+| ascending  | Sorted by a field or a function ascending.  | `asc`  | `SortDirection.ASCENDING`  |
+| descending | Sorted by a field or a function descending. | `desc` | `SortDirection.DESCENDING` |
+
+- Null ordering. It describes how to handle null values when ordering
+
+| Null ordering Type | Description                             | JSON          | Java                       |
+|--------------------|-----------------------------------------| ------------- | -------------------------- |
+| null_first         | Puts the null value in the first place. | `nulls_first` | `NullOrdering.NULLS_FIRST` |
+| null_last          | Puts the null value in the last place.  | `nulls_last`  | `NullOrdering.NULLS_LAST`  |
+
+Note: If the direction value is `ascending`, the default ordering value is `nulls_first` and if the direction value is `descending`, the default ordering value is `nulls_last`.
+
+- Sort term. It shows which field or function Gravitino uses to sort the table, please refer to the `Expression type` in the bucketed table section.
+
+<Tabs>
+<TabItem value="Json" label="Json">
+
+```json
+ {
+  "direction": "asc",
+  "nullOrder": "NULLS_LAST",
+  "sortTerm":  {
+    "type": "field",
+    "fieldName": ["score"]
+  }
+}
+```
+
+</TabItem>
+<TabItem value="java" label="Java">
+
+```java
+SortOrders.of(FieldReferenceDTO.of("score"), SortDirection.ASCENDING, NullOrdering.NULLS_LAST);
+```
+
+</TabItem>
+</Tabs>
+
+
+:::tip
+**Not all catalogs may support those features.**. Please refer to the related document for more details.
+:::
+
+The following is an example of creating a partitioned, bucketed table, and sorted order table:
+
+<Tabs>
+<TabItem value="shell" label="Shell">
+
+```shell
```

Comment:
Some of the doc I see using `bash`, I think you'd better unifying them all.

Reply:
I remember we use `bash` thorough the doc `manmage-metadata-using-gravitino.md`, maybe I should change them all.

Reply:
We use `shell` in other places, so better to change to `shell`.

Reply:
Using `bash` or `shell` may affect the code highlighting, although I am not entirely certain. I suggest that you take a look at the actual effect on the website.

Reply:
I have tested locally and it seems fine to use `shell`.